### PR TITLE
Initial implemetation of sqlalchemy orm

### PR DESCRIPTION
**Pull request by peopleAlreadyKnowWhoIAm**

### CODE REVIEW

File: requirements.txt
Comment (IgorBrat) [inline requirements.txt:1]: It's a good practice to put only libraries explicitely used in the project, not all the libraries installed along with them.

File: model/account.py
Comment (IgorBrat) [inline model/account.py:1]: That's might be a good idea to add custom string representation for debugging purposes (for all models from the task).

File: main.py
Comment (IgorBrat) [inline main.py:1]: Better to init and run app in method scope (if __name__ == "__main__"). But not mandatory,

**Review by IgorBrat — CHANGES_REQUESTED** [review]
Please look at suggestions and then merge.

File: main.py
Comment (peopleAlreadyKnowWhoIAm) [reply main.py:1]: Changed

File: model/account.py
Comment (peopleAlreadyKnowWhoIAm) [reply model/account.py:1]: Added in MappedAsDataclass as db.Model ancestor

File: requirements.txt
Comment (peopleAlreadyKnowWhoIAm) [reply requirements.txt:1]: Removed explicitly not required

**Comment by peopleAlreadyKnowWhoIAm** [comment]
@IgorBrat  please resolve comments if ok

**Review by IgorBrat — APPROVED** [review]

**Pull request merged**